config/crd/bases/dragonflydb.io_dragonflies.yaml

```diff
@@ -1130,6 +1129,11 @@ spec:
                       type: string
                     description: (Optional) Dragonfly Service Annotations
                     type: object
+                  labels:
+                    additionalProperties:
+                      type: string
+                    description: (Optional) Dragonfly Service Annotations
```

Comment:
Sorry I didn't point it earlier, can you change this too to match with the `Labels` comment ? I will merge once done.

Reply:
Sorry, fixed